core/background_workers.py

```
@@ -0,0 +1,437 @@
+    
+    # Get database pool
+    pool = await get_db_pool()
+    
+    # Read legacy configuration from environment (for backward compatibility)
+    abuse_threshold = int(os.getenv('ABUSE_THRESHOLD', 10))
+    abuse_interval = int(os.getenv('ABUSE_CHECK_INTERVAL', 60))
+    monitor_batch_size = int(os.getenv('MONITOR_BATCH_SIZE', 100))
+    monitor_interval = int(os.getenv('MONITOR_LOG_INTERVAL', 30))
+    
+    # Start legacy workers (if needed)
+    asyncio.create_task(abuse_detector_worker(abuse_threshold, abuse_interval))
```

Ruff (0.14.5)

Store a reference to the return value of `asyncio.create_task`

(RUF006)